modules/post/osx/gather/gitignore.rb

```diff
@@ -0,0 +1,38 @@
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Git Ignore Retriever',
+        'Description' => %q{
+          This module finds potentially sensitive items by finding .gitignore files.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'N!ght Jmp'],
+        'Platform' => [ 'osx' ],
```

Comment:
I guess this works on `unix` more so than just osx? 👀

Reply:
I would assume it works on either. I have just been targeting OSX for testing.